python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_build_distribution.py

```diff
@@ -31,7 +31,7 @@
 model = ot.SymbolicFunction(input_names, formulas)
 
 # %%
-# Then we create a sample `x` and compute the corresponding output sample `y`.
+# Then we create a sample :math:`x` and compute the corresponding output sample :math:`y`.
```

Comment:
I think that the symbol `x` is the name of the Python variable, not a mathematical variable here.
```suggestion
# Then we create a sample `x` and compute the corresponding output sample `y`.
```